src/test/java/org/openrewrite/java/testing/mockito/ReplacePowerMockitoIntegrationTest.java

```diff
@@ -630,4 +630,47 @@ public void testCalendarDynamic() throws Exception {
 """
         ));
     }
+
+    @Test
+    void powerMockitoCallsAreReplacedByMockitoCalls() {
+        //language=java
+        rewriteRun(java(
+          """
+            import org.junit.jupiter.api.BeforeEach;
+            import org.powermock.api.mockito.PowerMockito;
+            import java.util.Calendar;
+            
+            public class MyTest {
+            
+                private Calendar calendarMock;
+            
+                @BeforeEach
+                void setUp() {
+                    calendarMock = PowerMockito.mock(Calendar.class);
+                    PowerMockito.doCallRealMethod().when(calendarMock).getTime();
+                    PowerMockito.doNothing().when(calendarMock).clear();
+                    PowerMockito.doThrow(new NullPointerException()).when(calendarMock.getCalendarType());
+                }
+            }
+""",
```

Comment:
Java text blocks take the leading indentation of the final `"""`; might be best to align that with the other statements.
```suggestion
            """,
```